### ISSUE TITLE
Allow users to customize the cursor

### ISSUE THREAD

**Issue by abstractlyZach** [emersion/slurp#85]
Hi, I've been having trouble seeing the `slurp` cursor. I use a dark theme, so the base color of most of my windows are black. It looks like the `slurp` cursor is 1px black and has a 1px white border, which is really hard to see on top of black backgrounds for me. (Cursor is slightly above the center of the screen.)
![image](https://user-images.githubusercontent.com/7566043/116801317-89a89600-aabd-11eb-8830-e0dc674bf137.png)

I've been able to address the issue somewhat by increasing the opacity of the `slurp` background, but it's still a bit difficult. (Cursor is near the same position.)
![image](https://user-images.githubusercontent.com/7566043/116801314-844b4b80-aabd-11eb-8327-66208b5c8a87.png)

I would love to be able to change the color and thickness of both the cursor and cursor border! I think it's an easy win that will help a lot of people. 

I imagine it would look something like this:
```
OPTIONS
       -h
           Show help message and quit.

      ...
       -cc color
           Set cursor color. See COLORS for more detail.

       -cw weight
           Set cursor weight (line thickness).

       -cbc color
           Set cursor border color. See COLORS for more detail.

       -cbw weight
           Set the cursor border's weight (line thickness).
       ...
```

Happy to submit a PR if this is wanted.

**Comment by emersion** [comment]
I don't think that's easy to do, since the cursor image is coming from your cursor theme. We can't easily change how it looks.

I'd suggest fixing or changing your cursor theme.

**Comment by abstractlyZach** [comment]
Thanks @emersion :100: 

For anyone who comes through and is curious, I was able to solve my cursor problem!
![20210504_22h39m00s_grim](https://user-images.githubusercontent.com/7566043/117100935-e90ade00-ad29-11eb-9d0c-6cfb3f2ab916.png)

I'm using `sway` and `sway` uses client-side cursors. This means that in the case of most applications (including `slurp`), you just need to set a couple of environment variables. I was able to change the theme of my cursor and set it to a ridiculously large size like this:
```
XCURSOR_THEME=Adwaita XCURSOR_SIZE=100 slurp
```

These wikis helped me a lot:
https://wiki.archlinux.org/title/Sway#Change_cursor_theme_and_size
https://wiki.archlinux.org/title/Cursor_themes
